fw/bestroute.go

```diff
@@ -32,23 +32,23 @@ func (s *BestRoute) Instantiate(fwThread *Thread) {
 }
 
 // AfterContentStoreHit ...
-func (s *BestRoute) AfterContentStoreHit(pitEntry *table.PitEntry, inFace uint64, data *ndn.Data) {
+func (s *BestRoute) AfterContentStoreHit(pitEntry table.PitEntry, inFace uint64, data *ndn.Data) {
 	// Send downstream
 	core.LogTrace(s, "AfterContentStoreHit: Forwarding content store hit Data=", data.Name(), " to FaceID=", inFace)
 	s.SendData(data, pitEntry, inFace, 0) // 0 indicates ContentStore is source
 }
 
 // AfterReceiveData ...
-func (s *BestRoute) AfterReceiveData(pitEntry *table.PitEntry, inFace uint64, data *ndn.Data) {
-	core.LogTrace(s, "AfterReceiveData: Data=", data.Name(), ", ", len(pitEntry.InRecords), " In-Records")
-	for faceID := range pitEntry.InRecords {
+func (s *BestRoute) AfterReceiveData(pitEntry table.PitEntry, inFace uint64, data *ndn.Data) {
+	core.LogTrace(s, "AfterReceiveData: Data=", data.Name(), ", ", len(pitEntry.InRecords()), " In-Records")
```

Comment:
`PitEntry` is now an interface so we need to convert member variable accesses to member function accesses.